test/setup/ci-test.overide.yml

```diff
@@ -29,7 +33,8 @@ services:
       - AB_TESTING=true
```

Comment:
Now that we have a `NODE_ENV=true` added as well, can we refactor `AB_TESTING` to use that instead?

Reply:
Or is there a value in having 2 different testing flags?

Reply:
I don't think we need `NODE_ENV=test`. I think it was from something else I was trying.